oled_hijack_so.c

```diff
@@ -581,11 +576,11 @@ int sprintf(char *str, const char *format, ...) {
         strncmp(str, PWD0, sizeof(PWD0)-1) == 0))
     {
         // Cut SSID or password to prevent line break
-        va_start(args, format);
+        //va_start(args, format);
```

Comment:
Why this has been commented?

Reply:
I'm passing whole arguments as va_list, so no need for va_start and va_end